cake_thief/robertwpaul/cake_thief_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'minitest/autorun'
+require './cake_thief'
+
+class CakeThiefTest < Minitest::Test
+
+  def test_zero_capacity
+    cakes = [Cake.new(7, 160), Cake.new(3, 90), Cake.new(2, 15)]
+    capacity = 0
+    assert_equal 0, max_duffel_bag_value(cakes, capacity)
```

Comment:
Yeah, I think that's fair. I normally don't worry too much about test data that I think makes no difference to the outcome, but it's probably a good habit to worry a bit about it. Comes in handy once you start building larger test suites, with potentially expensive-to-create test data.